MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+global-exclude *.py

```

Comment:
Shouldn't this be `global-exclude *.pyc`?

Reply:
Good eye Keith.  I think you are correct.

Reply:
Actually, one other thing.  The global-exclude should appear last, not first.  The [order](https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands) of these patterns matters.